site/content/en/docs/concepts/fair_sharing.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Fair Sharing"
+date: 2025-05-28
+weight: 6
+description: >
+  Mechanisms in Kueue to share quota between tenants fairly.
+---
+
+
+### [Admission Fair Sharing](/docs/concepts/admission_fair_sharing)
+
+A mechanism for ordering workloads based on the historical resource usage of their source LocalQueues, giving preference to those that have consumed fewer resources over time.
+
+### [Preemption based Fair Sharing](/docs/concepts/preemption/#fair-sharing)
```

Comment:
As a follow-up, could you open another PR to have a dedicated Preemption-based Fair Sharing page?